demo/dist/index.html

```diff
@@ -1,14 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Are we supposed to keep this file?

Reply:
Yeah. I think eventually we'll add a build script that will copy this from the `src` directory?